### ISSUE TITLE
Misencoded Singlequotes & Linebreaks Invalidate JSON Exports

### ISSUE THREAD

**Issue by AngaBlue** [ItsBranK/DeveloperTools#2]
When JSON is a specified as the export format, fields with line breaks will invalidate the JSON format such as the Playlist Description of the Gridiron game mode.

```json
	{
		"Playlist Id": 46,
		"Playlist Player Count": 8,
		"Playlist Title": "Gridiron",
		"Playlist Description": "Huddle up and learn the rules of Gridiron:
- The standard Rocket League ball has been replaced with an American football in this 4v4 mode.
- Touching the ball attaches it to the roof of your car. Handoff to a teammate or pass it downfield by dodging. Double jumping drops the ball.
- Goals scored while the ball is attached are worth 7 points. Goals that are passed in or loose balls that score are worth 3 points. All own-goals are also worth 3 points.
- Stay in bounds! You'll fumble the ball if you cross the line marked on the Arena wall.
 Now hit the Gridiron and have fun!",
		"Playlist Ranked": false,
		"Playlist Image Url": "https://rl-cdn.psyonix.com/Playlists/Images/rl_nfl_gridiron2_ltm_playlist_bg.jpg",
		"Playlist Icon Active Url": "https://rl-cdn.psyonix.com/Playlists/Images/rl_ltm_gridiron_playlist_a.png"
	}
```

There also appears to be an issue when exporting descriptions in single quotes that don't encode correctly.

```json
	{
		"Playlist Id": 33,
		"Playlist Player Count": 6,
		"Playlist Title": "Spike Rush",
		"Playlist Description": "All players have Spikes which automatically engage after kickoff, allowing them to stick the ball to their car by driving into it. The ball carrier cannot use or collect boost, is easily demolished by opposing players, and can manually disengage their car's spikes to release the ball. Score by getting the ball into your opponents goal by any means.",
		"Playlist Ranked": false,
		"Playlist Image Url": "https://rl-cdn.psyonix.com/Playlists/Images/rl_event_mode_bg_spikerush.jpg",
		"Playlist Icon Active Url": "https://rl-cdn.psyonix.com/Playlists/Images/rl_ltm_spikerush_playlist.png"
	}
```
`Score by getting the ball into your opponents goal by any means.`

**Comment by ItsBranK** [comment]
Thanks for the info about the line breaks, the second one I fully blame Psyonix on. If you look at the static.json contents for that very last "quote" is actually a `’` unicode character (U+2019) when its suppose to be a `'` character (039 in ASCII).  That specific UTF16 char doesn't have an equivalent version of it in ASCII without replacing it.

**Comment by AngaBlue** [comment]
Yep no worries.  Would you consider replacing that apostrophe and other occurrences in your export or just stay true to the game?

**Comment by ItsBranK** [comment]
Both fixed in v3.4
